autogen/coding/docker_commandline_code_executor.py

```diff
@@ -39,6 +39,31 @@ def _wait_for_ready(container: Any, timeout: int = 60, stop_time: float = 0.1) -
 
 
 class DockerCommandLineCodeExecutor(CodeExecutor):
+    SUPPORTED_LANGUAGES: ClassVar[List[str]] = [
+        "bash",
+        "shell",
+        "sh",
+        "pwsh",
+        "powershell",
+        "ps1",
+        "python",
+        "javascript",
+        "html",
+        "css",
+    ]
+    DEFAULT_EXECUTION_POLICY: ClassVar[Dict[str, bool]] = {
+        "bash": True,
+        "shell": True,
+        "sh": True,
+        "pwsh": True,
+        "powershell": True,
+        "ps1": True,
+        "python": True,
+        "javascript": False,
+        "html": False,
+        "css": False,
+    }
```

Comment:
Are these two mirrors of each other? If so, we can just keep `DEFAULT_EXECUTION_POLICY` and assume every language that's in this dictionary is a supported language

Reply:
Good suggestion, I simplify the implementation using only the default_policy